helix-term/src/ui/menu.rs

```diff
@@ -205,16 +205,16 @@ impl<T: Item + 'static> Component for Menu<T> {
             _ => return EventResult::Ignored(None),
         };
 
-        let close_fn = EventResult::Consumed(Some(Box::new(|compositor: &mut Compositor, _| {
+        let close_fn = Some(Box::new(|compositor: &mut Compositor, _| {
```

Comment:
I think you can make the error go away by helping the type inference here: `let close_fn: Callback = ...`

https://github.com/helix-editor/helix/blob/038aaa1b3f46d478875b4c85a1382aa12226f70a/helix-term/src/compositor.rs#L10